tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_RTL_POC.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.settings.app import Settings
+from marionette_driver import By
+
+
+class TestSettingsRTL(GaiaImageCompareTestCase):
+    _page_locator = (By.ID, 'root')
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+
+    def test_settings_app(self):
+
+        settings = Settings(self.marionette)
+        settings.launch()
+
+        self.take_screenshot('main')
+
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+
+        settings.open_wifi_settings()
```

Comment:
Let's remove the duplication.
``` python
for subpage in ('wifi', 'sim_manager', ...):
   open_method = settings.__getattribute__('open_{}_settings'.format(subpage))
   open_method()
   self.take_screenshot(subpage)
   settings.return_to_main_menu()
```

Side note: `settings.open_*_settings` sounds redundant. @npark-mozilla how do you feel about just call these method `open_*`? (It might be better to do it in a follow up)